data/sql/updates/pending_db_world/rev_1584283998126323200.sql

```diff
@@ -0,0 +1,24 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1584283998126323200');
+
+-- Grizzly Hills, Alliance Log Ride Start
+-- npc gordun
+DELETE FROM `gossip_menu_option` WHERE `MenuID`=21251 AND `OptionID`=1;
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (21251, 1, 0, 'Yes, I am ready to travel to Venture Bay!', 26605, 1, 1, 0, 0, 0, 0, NULL, 0, 0);
```

Comment:
```suggestion
INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES 
(21251, 1, 0, 'Yes, I am ready to travel to Venture Bay!', 26605, 1, 1, 0, 0, 0, 0, NULL, 0, 0);
```

Reply:
Just for consistency and easier to review the SQL this way imo.

Great job with this!